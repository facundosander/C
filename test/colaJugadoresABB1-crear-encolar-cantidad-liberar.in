crearCola

# imprimo la cantidad de elementos de la cola, que debe ser 0
imprimirCantidadCola

# Creo arbol con Jugadores 4 y 5
crearArbolJugadores

crearJugador 5 11 Argentina Maria
insertarJugadorArbolJugadores

crearJugador 4 21 Uruguay Juan
insertarJugadorArbolJugadores

# Encolo el arbol con Jugadores 4 y 5
encolar

# imprimo la cantidad de elementos de la cola, que debe ser 1
imprimirCantidadCola

# obtengo el frente de la cola, que es el árbol con los Jugadores 4 y 5
obtenerFrenteCola

# libero la memoria asociada a la cola
liberarCola

# imrpimo el arbol con los Jugadores 4 y 5
imprimirArbolJugadores

# libero el árbol
liberarArbolJugadores

Fin