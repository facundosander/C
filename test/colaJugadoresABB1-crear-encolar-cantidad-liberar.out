1>2>#  imprimo la cantidad de elementos de la cola, que debe ser 0.
3>La cantidad de arboles de jugadores en la cola es: 0.
4>#  Creo arbol con Jugadores 4 y 5.
5>6>7>8>9>10>#  Encolo el arbol con Jugadores 4 y 5.
11>12>#  imprimo la cantidad de elementos de la cola, que debe ser 1.
13>La cantidad de arboles de jugadores en la cola es: 1.
14>#  obtengo el frente de la cola, que es el árbol con los Jugadores 4 y 5.
15>Jugador 4:  Juan, Uruguay, 21 años
Jugador 5:  Maria, Argentina, 11 años
16>#  libero la memoria asociada a la cola.
17>18>#  imrpimo el arbol con los Jugadores 4 y 5.
19>Jugador 4:  Juan, Uruguay, 21 años
Jugador 5:  Maria, Argentina, 11 años
20>#  libero el árbol.
21>22>Fin.
